core/services/pipeline/task.bridge.go

```diff
@@ -217,7 +220,7 @@ func (t *BridgeTask) Run(ctx context.Context, lggr logger.Logger, vars Vars, inp
 	}
 
 	if !cachedResponse && cacheTTL > 0 {
-		err := t.orm.UpsertBridgeResponse(ctx, t.dotID, t.specId, responseBytes)
+		err := t.orm.UpsertBridgeResponse(overtimeCtx, t.dotID, t.specId, responseBytes)
```

Comment:
Looks fine, but just wanted to note something I noticed in the logs in case it might be relevant to how we fix this one:  there were timeouts for both of these methods, but for some reason the timeout printed in `UpsertBridgeResponse` was always positive (I remember seeing hundreds of milliseconds left on the clock), while for `GetCacheResponse` it was always 0 or negative. Any idea why that difference?

Reply:
Were the Upsert logs failures, or just timing logs?

Reply:
They are SLOW QUERY messages... but they got logged at log level WARN instead of CRIT like the other ones:
```
2024-05-31T00:42:08.724Z [WARN]  SLOW SQL QUERY                                     bridges/orm.go:200               logger=WrappedDB ms=96 percent=10.8 sql=INSERT INTO bridge_last_value(dot_id, spec_id, value, finished_at)      VALUES('ds1', 328, '\x7b22726573756c74223a302e      303432383139343939333033323232352c2264617461223a7b22726573756c74223a302e303432383139343939333033323232357d2c2274696d657374616d7073223a7b2270726f76696465724461746153747265616d45737461626c6973686564556e69784d73223a313731363838393434343733352c2270726f766964657244617461526563656      9766564556e69784d73223a313731373131363132373935372c2270726f7669646572496e6469636174656454696d65556e69784d73223a313731373131363132363030307d2c22737461747573436f6465223a3230302c226d657461223a7b22616461707465724e616d65223a2243525950544f434f4d50415245222c226d657472696373223a7b22      666565644964223a227b5c22626173655c223a5c22616e6b725c222c5c2271756f74655c223a5c227573645c227d227d7d7d', 2024-05-31 00:42:08.627802951 +0000 UTC m=+450.814588173)    ON CONFLICT ON CONSTRAINT bridge_last_value_pkey     DO UPDATE SET value = '\x7b22726573756c74223a302e303432383      139343939333033323232352c2264617461223a7b22726573756c74223a302e303432383139343939333033323232357d2c2274696d657374616d7073223a7b2270726f76696465724461746153747265616d45737461626c6973686564556e69784d73223a313731363838393434343733352c2270726f766964657244617461526563656976656455      6e69784d73223a313731373131363132373935372c2270726f7669646572496e6469636174656454696d65556e69784d73223a313731373131363132363030307d2c22737461747573436f6465223a3230302c226d657461223a7b22616461707465724e616d65223a2243525950544f434f4d50415245222c226d657472696373223a7b22666565644      964223a227b5c22626173655c223a5c22616e6b725c222c5c2271756f74655c223a5c227573645c227d227d7d7d', finished_at = 2024-05-31 00:42:08.627802951 +0000 UTC m=+450.814588173; timeout=891 version=2.12.0@a06b2f2
```

I guess the logger that handles printing SLOW SQL QUERY messages must decide on the severity of the log level based on how much time is left, and if it's <= 0 it uses CRIT?  That makes sense I guess.

Reply:
So I guess I was wrong when I said "there were timeouts"... this wasn't a timeout in the sense that, the time limit was never reached... just came closer than was expected for some reason.